CHANGELOG.md

```diff
@@ -7,7 +7,29 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 <!-- towncrier release notes -->
 
-## [v0.104.0-sumo-1]
+## [v0.106.1-sumo-0]
```

Comment:
We should update the tag to v0.106.1-sumo-1

Reply:
This PR needs a new change log, stating that we're fixing the changelog of the previous 106 release